packages/common/src/authentication/AuthContext.tsx

```diff
@@ -9,6 +9,12 @@ import { useGetAuthToken } from './api/hooks/useGetAuthToken';
 import { AuthenticationResponse } from './api';
 
 export const COOKIE_LIFETIME_MINUTES = 60;
+const TOKEN_CHECK_INTERVAL = 60 * 1000;
+
+export enum AuthError {
+  PermissionDenied = 'PermissionDenied',
```

Comment:
No idea what this will look like as an api response - should be easy to add in when the api implements though